aptos-move/move-examples/event/sources/event.move

```diff
@@ -0,0 +1,54 @@
+/// This provides an example shows how to use module events.
+
+module event::event {
+    use aptos_framework::event;
+    #[test_only]
+    use std::vector;
+
+    struct Field has store, drop {
+        field: bool,
+    }
+
+    #[event]
+    struct MyEvent has store, drop {
+        seq: u64,
+        field: Field,
+        bytes: vector<u64>
+    }
+
+    public entry fun emit(num: u64) {
+        let i = 0;
+        while (i < num) {
+            let event = MyEvent {
+                seq: i,
+                field: Field { field: false },
+                bytes: vector[]
+            };
+            event::emit(&event);
```

Comment:
Can you add a quick test/function for emitting events in an inline function as well?